packages/react-instantsearch-core/src/connectors/connectRange.js

```diff
@@ -65,37 +65,32 @@ function getCurrentRange(boundaries, stats, precision) {
 }
 
 function getCurrentRefinement(props, searchState, currentRange, context) {
-  const refinement = getCurrentRefinementValue(
+  let refinement = getCurrentRefinementValue(
```

Comment:
We can inline the destruct assignment that we do [L76](https://github.com/algolia/react-instantsearch/pull/2350/files#diff-588750eca1a5cfabf9bda1e9070350ebR76).